src/main/java/uk/gov/hmcts/divorce/caseworker/service/ReIssueApplicationService.java

```diff
@@ -72,6 +72,8 @@ public CaseDetails<CaseData, State> process(final CaseDetails<CaseData, State> c
         updatedCaseDetails.getData().getApplication().setPreviousReissueOption(reissueOption);
         updatedCaseDetails.getData().getApplication().setReissueOption(null);
 
+        updatedCaseDetails.getData().getAcknowledgementOfService().setAosIsDrafted(null);
```

Comment:
why do we need to make this field null ? you can still use Update aos ? I think the first question will caseworker reissue after drafting aos ?